kuka_lbr_iiwa_support/package.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0"?>
+<package>
+  <name>kuka_lbr_iiwa_support</name>
+  <version>0.0.0</version>
+  <description>
+    <p>
+      ROS-Industrial support for the KUKA LBR IIWA (and variants).
+    </p>
+    <p>
+      This package contains configuration data, 3D models and launch files
+      for KUKA LBR IIWA manipulators. This currently includes the base model.
+    </p>
+    <p>
+      Joint limits and max joint velocities are based on the information in
+      the KUKA data sheets.  All URDFs / XACROs are based on the
+      default motion and joint velocity limits, unless noted otherwise (ie:
+      no support for high speed joints, extended / limited motion ranges or
+      other options).
+    </p>
+    <p>
+      Before using any of the configuration files and / or meshes included
+      in this package, be sure to check they are correct for the particular
+      robot model and configuration you intend to use them with.
+    </p>
+  </description>
+  <author email="levi.armstrong@swri.org">Levi Armstrong (Southwest Research Institute)</author>
+  <maintainer email="jon@todo.todo">jon</maintainer>
+
+  <license>Apache 2.0</license>
```

Comment:
I've seen a lot of pkgs having just `Apache2` here. Not sure what is actually 'correct'. [REP-140 - license](http://www.ros.org/reps/rep-0140.html#license-multiple-but-at-least-one) gives `Apache 2.0`.